wordgenerator.py

```diff
@@ -0,0 +1,106 @@
+import pygame
+import random
+import sys
+import math
+from copy import deepcopy
+
+CONST_FILENAME = "american-english.txt"  
+
+'''
+Opens the file CONST_FILENAME and reads entire file to word_array list. Shuffles the array and then sorts according to length
+'''
+class WordGenerator():
+    index_locations = []
+    index_prime_locations = []
+    word_array = []
+    number_of_lengths = 0
+    smallest_length = []
+
+    def __init__(self):
+        with open(CONST_FILENAME) as infile:
+            for line in infile:
+                self.word_array.append(line.rstrip('\n'))
+        random.shuffle(self.word_array) 
+        self.word_array.sort(key=len)
```

Comment:
So in this line, did you test that if they have the same len they'll still be random?